app/services/idv/flows/in_person_flow.rb

```diff
@@ -4,8 +4,8 @@ class InPersonFlow < Flow::BaseFlow
       STEPS = {
         location: Idv::Steps::Ipp::LocationStep,
         welcome: Idv::Steps::Ipp::WelcomeStep,  # instructions
-        address: Idv::Steps::Ipp::AddressStep,  # entering the address
         state_id: Idv::Steps::Ipp::StateIdStep, # info from state id
+        address: Idv::Steps::Ipp::AddressStep,  # entering the address
```

Comment:
Just wanting to double-check: Referencing the Figma designs, it has the address step first. Is this an intentional change?

Reply:
We decided to switch the order of the pages; it's an AC (#4) in [the ticket](https://cm-jira.usa.gov/browse/LG-6090), but isn't reflected in the Figma yet.

Reply:
Waiting for Julia's approval on this, and then will update Figma